### ISSUE TITLE
GET sale/products - informacje o popularności całego produktu z katalogu allegro

### ISSUE THREAD

**Issue by Daczek** [allegro/allegro-api#9776]
Witam,
Używam funkcji GET sale/products do wyszukania pasujących produktów z katalogu allegro. Kiedy już mam kilka ID produktów z katalogu allegro chciałbym porównać ich popularność. Czy jest to w jakikolwiek sposób możliwe?

Np czy mogę pobrać za pomocą GET /offers/listing wszystkie produkty przypisane do danego ID produktu z katalogu allegro? Albo w jakikolwiek inny sposób sprawdzić popularność danego produktu z katalogu allegro?

Pozdrawiam.

**Comment by MaciejFrackowiak** [comment]
Nie, nie sprawdzisz w ten sposób popularności produktów. Nie udostępniamy też osobnego endpointu do informowania o popularności danego produktu.
